cdpfn.go

```diff
@@ -237,6 +238,62 @@ var CDPFnMap = map[string]CDPFn{
 		},
 		Aliases: []string{"eval"},
 	},
+	"localStorage": {
+		Desc: "Get localStorage items.",
+		Fn: func(origin string, items *map[string]string) chromedp.Action {
+			return chromedp.ActionFunc(func(ctx context.Context) error {
+				storageID := &domstorage.StorageID{
+					SecurityOrigin: origin,
+					IsLocalStorage: true,
+				}
+				resp, err := domstorage.GetDOMStorageItems(storageID).Do(ctx)
+				if err != nil {
+					return err
+				}
+				m := make(map[string]string)
+				for _, v := range resp {
+					if len(v) != 2 {
+						continue
+					}
+					m[v[0]] = v[1]
+				}
+				*items = m
+				return nil
+			})
+		},
+		Args: CDPFnArgs{
+			{CDPArgTypeArg, "origin", "https://github.com"},
+			{CDPArgTypeRes, "items", `{"key": "value"}`},
+		},
+	},
+	"sessionStorage": {
+		Desc: "Get sessionStorage items.",
+		Fn: func(origin string, items *map[string]string) chromedp.Action {
+			return chromedp.ActionFunc(func(ctx context.Context) error {
+				storageID := &domstorage.StorageID{
+					SecurityOrigin: origin,
+					IsLocalStorage: false,
+				}
+				resp, err := domstorage.GetDOMStorageItems(storageID).Do(ctx)
+				if err != nil {
+					return err
+				}
+				m := make(map[string]string)
+				for _, v := range resp {
+					if len(v) != 2 {
+						continue
+					}
+					m[v[0]] = v[1]
+				}
+				*items = m
+				return nil
+			})
+		},
+		Args: CDPFnArgs{
+			{CDPArgTypeArg, "origin", "https://github.com"},
+			{CDPArgTypeRes, "items", `{"key": "value"}`},
+		},
+	},
```

Comment:
Please add alias. 
You can update the documentation with `make doc`.

```suggestion
		Aliases: []string{"getSessionStorage"},
	},
```